frontend/src/metabase-lib/Question.ts

```diff
@@ -443,6 +443,13 @@ class QuestionInner {
     return this.query().isEmpty();
   }
 
+  /**
+   * How many filters are this question's values used for?
+   */
+  filterValuesUsageCount(): number {
+    return this.card().filter_values_usage_count || 0;
```

Comment:
There's a slight footgun here which is that `filter_values_usage_count` is only hydrated in GET `/api/card/:id`. As far as I know that's the only place that's relevant, but maybe a frontender can confirm that we won't ever have a card lying around that came from some other endpoint.

Reply:
There's a function naming convention that's not explicitly required in our FE codebase, but we see it in use and it's helpful:

the first word in a function name is a verb.

So it'll often be the main idea of what the function will do.

In this case, seeing `filter` first, I thought this function might filter something.

If we want to stick with the usage, maybe something like `getFilterCountForValues`, or `getFilterCountForQuestionValues`.  I'd prefer the first as the call will probably be something like `question.getFilterCountForValues` and mention questions anyway.

Reply:
Makes sense, just renamed it. Thanks!